include/psa/crypto_config.h

```diff
@@ -1665,15 +1665,15 @@
  *
  * Uncomment this macro to enable restartable ECC computations.
  */
-//#define MBEDTLS_ECP_RESTARTABLE
+// #define MBEDTLS_ECP_RESTARTABLE
 
 /**
  * Uncomment to enable using new bignum code in the ECC modules.
  *
  * \warning This is currently experimental, incomplete and therefore should not
  * be used in production.
  */
-//#define MBEDTLS_ECP_WITH_MPI_UINT
+// #define MBEDTLS_ECP_WITH_MPI_UINT
```

Comment:
```suggestion
//#define MBEDTLS_ECP_WITH_MPI_UINT
```

Reply:
Thanks, I dropped the commit containing those changes as that is all it appeared to have.